semgrep-core/src/analyzing/Dataflow.ml

```diff
@@ -46,7 +46,10 @@ module type Flow = sig
 
   type edge
 
-  type flow = (node, edge) Ograph_extended.ograph_mutable
+  type flow = {
+    graph : (node, edge) Ograph_extended.ograph_mutable;
+    entry : int;
```

Comment:
Use the `nodei` type alias here rather than `int`.

Reply:
Hmm, maybe better to define a type previously,

```
type ('node, 'edge) cfg = {
     graph : ('node, 'edge) Ograph_extended.ograph_mutable;
     entry : nodei;
}
```
And then refer to this everywhere else, so here it would be:
```
type flow = (node, edge) cfg;
```